app/helpers/application_helper.rb

```diff
@@ -68,6 +68,16 @@ def encoded_page_url
     World Database on Protected Areas (WDPA), and the most comprehensive
     global database on terrestrial and marine protected areas.
   """
+
+  def yml_key
+    case controller_name
+    when 'target_dashboard'
+      'thematic_area.target_11_dashboard'
+    else 
+      nil
+    end
+  end
```

Comment:
While this is working fine, I would refactor using the ternary operator rather than a `case` statement, so something like:
`controller_name = 'target_dashboard' ? 'thematic_area.target_11_dashboard' : nil`

Reply:
Oh, I see. Let's keep it as it is then.